src/extensions/mvs/load.ts

```diff
@@ -152,6 +153,13 @@ const MolstarLoadingActions: LoadingActions<MolstarTree, MolstarLoadingContext>
         });
         return model;
     },
+    raw_volume(updateParent: UpdateTarget, node: SubTreeOfKind<MolstarTree, 'raw_volume'>, context: MolstarLoadingContext): UpdateTarget {
+        // TODO: write map metadata to the context
+        const props = rawVolumeProps(node);
+        const rawVolume = UpdateTarget.apply(updateParent, VolumeFromCcp4, props);
+        // TODO: tooltips, labels, etc.
+        return rawVolume;
+    },
```

Comment:
I'm not sure if different values of `node.options.source` will affect implementation here.
If so, pls include a check here or in `rawVolumeProps`, so that once we add more allowed values for `source`, it will throw error instead of trying to apply wrong implementation.

Reply:
Yes, you are right, currently the different values of `node.options.source` will indeed not affect the implementation. I have added the `node.options.source` attribute since in we potentially may add support for other sources of volumetric data in the future, e.g., for OMEZarr. In that case, we will need other transforms or mechanisms for parsing them and transforming to the Mol* `Volumes`.

Sure, will add a check here.